### DISCUSSION TITLE
Recurring meals in meal plan

### DISCUSSION THREAD

**Discussion by TheEpicTen** [mealie-recipes/mealie#4525] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Ability to add recurring meals to the meal planner.

For example, adding the recipe “Pizza” to be on the meal planner every Friday would be very useful.

### Please provide a concise description of the feature that would resolve your issue.

Recurring meals in meal plan. See above.

### Please consider and list out some caveats or tradeoffs made in your design decision

N/A

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by saraheary19** [comment]
Agreed. This would be nice to reduce the need to individually select reoccuring which for me would be more like breakfast or lunch or days that are eat out every week.

**Comment by t3rr0rz0n3** [comment]
This would be very useful. For example, I have two weekly meal plans that I alternate between. One week I have a specific set of lunches and dinners, and the following week I have a different set. Then the cycle repeats.

This way of organizing things works really well for me because I can use each weekly meal plan to generate the shopping list for that week.

It would be great to be able to create rules such as: “Every Monday, add this meal for lunch/dinner/breakfast/etc.” If you configured a rule for every day, you could automatically end up with a full monthly calendar containing all your meals.

Even better, if these rules could repeat indefinitely, the following month would automatically follow the same pattern, so there would be no need to manually fill in the meal plan every week.
